src/dlr_tensorflow2/dlr_tensorflow2.cc

```diff
@@ -128,7 +128,7 @@ void Tensorflow2Model::PrepInputs() {
   for (std::string& t_name : input_tensor_names_) {
     TF_Output oper_out = ParseTensorName(t_name);
     const TF_DataType t_type = TF_OperationOutputType(oper_out);
-    input_types_.push_back(std::to_string((int)t_type));
+    input_types_.push_back(tensorflow::DataType_Name(static_cast<tensorflow::DataType>(t_type)));
```

Comment:
instead of `static_cast` we can get input/output type from `tensorflow::TensorInfo`. See https://github.com/neo-ai/neo-ai-dlr/pull/428/files#diff-94c6e4f63ba097968d30f0f54346e0a8493faa0ed8f2766489e4cadfdaf77cee